SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/ReferenceCountedLockingPolicyTest.cpp

```diff
@@ -216,27 +260,22 @@ REFERENCECOUNTEDLOCKINGPOLICY_TEST(GIVEN_an_actor_has_not_been_locked_WHEN_IsLoc
 	ADD_LATENT_AUTOMATION_COMMAND(FWaitForWorld(Data));
 	ADD_LATENT_AUTOMATION_COMMAND(FSpawnActor(Data, "Actor"));
 	ADD_LATENT_AUTOMATION_COMMAND(FWaitForActor(Data, "Actor"));
-	ADD_LATENT_AUTOMATION_COMMAND(FTestIsLocked(this, Data, "Actor", false, 0));
+	ADD_LATENT_AUTOMATION_COMMAND(FTestIsLocked(this, Data, "Actor", false));
 
 	return true;
 }
 
-REFERENCECOUNTEDLOCKINGPOLICY_TEST(GIVEN_AcquireLock_is_called_WHEN_IsLocked_is_called_THEN_returns_true_with_one_lock_token)
+REFERENCECOUNTEDLOCKINGPOLICY_TEST(GIVEN_Actor_is_not_locked_WHEN_ReleaseLock_is_called_THEN_it_errors_and_returns_false)
```

Comment:
Doesn't have to be in this PR, but would love to see a test where the locked actor is deleted whilst locked.

Reply:
Nice, I also want to do tests to fail acquiring locks for non-authoritative Actors, I'll make a ticket

Reply:
https://improbableio.atlassian.net/browse/UNR-2831